src/client-app/src/models/Pieces/Helper/MoveHelper.ts

```diff
@@ -0,0 +1,31 @@
+import { Game } from "../../game";
+import { Coordinate } from "../../chess";
+
+export class MoveHelper {
```

Comment:
Is there a reason for this to be in a class? I was thinking if `checkDirection` could just be a standalone function because it does not require any class-specific logic 😅.